src/blocks.js

```diff
@@ -11,8 +14,69 @@ export default (editor, opts = {}) => {
   } else if ('grapesjsnewsletter' in cfg.pluginsOpts) {
     const dcb = new DynamicContentBlocks(editor, opts);
     dcb.addDynamciContentBlock();
+  }else if ('grapesjswebpage' in cfg.pluginsOpts) {
```

Comment:
I think this can be solved more nicely now with the new function: ContentService.getMode(editor) 

(the same is true for the existing code on one 14.)